projects/canopy/src/lib/carousel/carousel.component.ts

```diff
@@ -14,11 +14,12 @@ import { filter, map, takeUntil, withLatestFrom } from 'rxjs/operators';
 import { NgIf, NgFor, NgClass } from '@angular/common';
 
 import type { HeadingLevel } from '../heading';
-import { LgIconComponent } from '../icon';
+import { LgIconComponent, lgIconChevronLeft, lgIconChevronRight } from '../icon';
 import { LgHeadingComponent } from '../heading';
 
 import { LgCarouselItemComponent } from './carousel-item/carousel-item.component';
 import { LgAutoplayComponent } from './auto-play/auto-play.component';
+import { LgIconRegistry } from './../icon/icon.registry';
```

Comment:
I'm surprised the linter didn't complain about this; the `../` paths should all be grouped together. I think you tricked it by starting with `./` :D

Reply:
ohhh, I used auto import, I will fix this

Reply:
I've fixed this

Reply:
I thought what Paul meant was move it to line 18 and remove the `./` so you have `import { LgIconRegistry } from '../icon/icon.registry';` but i might be wrong!